### PR TITLE
[QoS]Increasing LACP timer for lag ports for broadcom-dnx neighbor EOS host   

### PR DESCRIPTION
**Pull request by ansrajpu-git**
### Description of PR

Intermittently testQosSaiLossyQueue tests fails due to Port-channel flap on broadcom-dnx T2 Voq chassis.
The reason the port-channel goes down is because this test requires disabling TX on the egress port (which is a member of a port-channel)
With the huge buffer-size, it takes a longer time to send packets . This will result in the TX LACP packets to stop egressing, so after 3 LACP packets are missed (~90s) on the server side the LAG is torn down.

Issue # https://github.com/sonic-net/sonic-mgmt/issues/11682
Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [x] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
Intermittently testQosSaiLossyQueue tests fails due to Port-channel flap
#### How did you do it?
The lacp timer multiplier on the EOS host is configurable.
By default, timeout is 30 secs with a failure tolerance of 3. 
We changed the multiplier to an increased value to hold the connectivity for some time until all packets are sent.
And revert the changes after test case execution.

#### How did you verify/test it?
Executed qos test cases and verfiy the results.

### CODE REVIEW

**Comment by ansrajpu-git** [comment]
@vmittal-msft .Please review.

**Comment by abdosi** [comment]
can we run this test on dest being non portchanne ?

**Comment by arlakshm** [comment]
@ansrajpu-git, can you fix the pre-commit failures?

**Comment by ansrajpu-git** [comment]
@arlakshm, please review all checked passed now.

**Comment by vmittal-msft** [comment]
@ansrajpu-git Looks like this change won't work if fanout is non EOS ?

**Comment by ansrajpu-git** [comment]
> @ansrajpu-git Looks like this change won't work if fanout is non EOS ?

@vmittal-msft , setting Lacp timer is not supported for vsonic neighbors

**Comment by vmittal-msft** [comment]
@ansrajpu-git how about any other HW vendor fanout switches?

**Comment by ansrajpu-git** [comment]
> @ansrajpu-git how about any other HW vendor fanout switches?

@vmittal-msft , For EOS based VMs, these changes should work. For vsonic based VMs, we are skipping the test. These changes are only for broadcom-dnx platform.

**Review by vmittal-msft — APPROVED** [review]

**Review by arlakshm — APPROVED** [review]

**Comment by arlakshm** [comment]
@yejianquan can you please approve this change for 202405. This changes is for DNX platforms only

File: tests/qos/qos_sai_base.py [hunk 1]
```diff
@@ -2572,3 +2573,49 @@ def isLonglink(self, dut_host):
...
+                              request):
+        if request.config.getoption("--neighbor_type") == "sonic":
+            yield
+            return
+
+        if ('platform_asic' in dutTestParams["basicParams"] and
+                dutTestParams["basicParams"]["platform_asic"] == "broadcom-dnx"):
+            src_dut = get_src_dst_asic_and_duts['src_dut']
+            dst_dut = get_src_dst_asic_and_duts['dst_dut']
+            if src_dut.sonichost.is_multi_asic and dst_dut.sonichost.is_multi_asic:
```
Comment (arista-nwolfe) [inline tests/qos/qos_sai_base.py:2588]: What's the reason we only want to do this increase on multi-asic systems? The LACP timeout issue is a problem on single-asic systems as well right?

**Comment by bingwang-ms** [comment]
Pending on #15778

**Comment by sdszhang** [comment]
@Javier-Tan this is the prerequisite for #16224, can you cherry pick this into 202405-msft? Thanks.

@rlhui @vmittal-msft for viz.

**Comment by Javier-Tan** [comment]
> @Javier-Tan this is the prerequisite for #16224, can you cherry pick this into 202405-msft? Thanks.
> 
> @rlhui @vmittal-msft @yejianquan for viz.

Ack

**Comment by yejianquan** [comment]
only for chassis, only go to 202405-msft rather than 202405

**Pull request merged**